pkg/appconsts/appconsts.go

```diff
@@ -40,22 +40,22 @@ const (
 	// blob) share.
 	SparseShareContentSize = ShareSize - NamespaceSize - ShareInfoBytes
 
-	// MaxSquareSize is the maximum number of
+	// DefaultMaxSquareSize is the maximum number of
 	// rows/columns of the original data shares in square layout.
 	// Corresponds to AVAILABLE_DATA_ORIGINAL_SQUARE_MAX in the spec.
 	// 128*128*256 = 4 Megabytes
 	// TODO(ismail): settle on a proper max square
 	// if the square size is larger than this, the block producer will panic
-	MaxSquareSize = 128
+	DefaultMaxSquareSize = 128
 	// MaxShareCount is the maximum number of shares allowed in the original data square.
 	// if there are more shares than this, the block producer will panic.
-	MaxShareCount = MaxSquareSize * MaxSquareSize
+	MaxShareCount = DefaultMaxSquareSize * DefaultMaxSquareSize
 
-	// MinSquareSize depicts the smallest original square width. A square size smaller than this will
+	// DefaultMinSquareSize depicts the smallest original square width. A square size smaller than this will
 	// cause block producer to panic
-	MinSquareSize = 1
+	DefaultMinSquareSize = 1
 	// MinshareCount is the minimum shares required in an original data square.
-	MinShareCount = MinSquareSize * MinSquareSize
+	MinShareCount = DefaultMinSquareSize * DefaultMinSquareSize
```

Comment:
[do in a different PR]

we could even put these in their own blob file as this one is getting crowded